sudoqler.py

```diff
@@ -21,11 +21,11 @@ def from_one_line(one_line_sudoku):
     """
     length = len(one_line_sudoku)
     if length > puzzle_size:
-        warning_body = ["Too many values: Needed {}, but received {}",
-                        "Ignored values: {} ..."]
-        warnings.warn('\n'.join(warning_body).format(puzzle_size, length,
-            one_line_sudoku[puzzle_size:puzzle_size + 7]
-            .encode('string_escape')))
+        ignored_values = one_line_sudoku[puzzle_size:].encode()
+        if ignored_values == '\n':
+            warning_body = ["Too many values: Needed {}, but received {}",
+                            "Ignored values: {} ..."]
+            warnings.warn('\n'.join(warning_body).format(puzzle_size, length, ignored_values))
```

Comment:
Had to remove `encode('string_escape')` for 3, but otherwise, just tidying up and handling typical trailing newline.